<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Image/Icon/setting.png</normaloff>:/Image/Icon/setting.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxThemdaucham">
     <property name="text">
      <string>Thêm dấu chấm vào cuối đoạn văn nếu chưa có dấu . ! ?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBoxLuidaudong">
       <property name="text">
        <string>Lùi đầu dòng (dấu cách) :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxdaucach">
       <item>
        <property name="text">
         <string>1 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9 dấu cách</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 dấu cách</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxThaythetuviettat">
     <property name="text">
      <string>Tự động thay thế các từ viết tắt.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxLoikieugo">
     <property name="text">
      <string>Tự động sửa lỗi do kiểu gõ.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonXacnhan">
       <property name="text">
        <string>Xác nhận</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHuybo">
       <property name="text">
        <string>Huỷ bỏ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
